gkeys/gkeys/keyhandler.py

```diff
@@ -108,3 +111,34 @@ def key_search(self, args, first_match=False):
 
         self.logger.debug(_unicode("KeyHandler: key_search; keys = %s") % str(keys))
         return keys
+
+    def is_expiring(self, keys, days_limit=30):
```

Comment:
the is_expiring function still does not use staticmethod, check dol-sen's previous comments on your PR